install/kubernetes/cilium/charts/hubble-ui/values.yaml

```diff
@@ -5,7 +5,7 @@ image:
   # tag is the container image tag to use.
   # Ref: https://github.com/cilium/hubble-ui/releases
   # Ref: https://quay.io/repository/cilium/hubble-ui?tab=tags
-  tag: v0.5.0
+  tag: latest
```

Comment:
What's the reason for this change? I'm vary of re-introducing the latest tag. But if we do, please also change the default pull policy to Always.

Reply:
this is because `latest` is the only image working in 1.8.0-rc1 with hubble-relay enabled. Note that hubble-ui v0.5.0 also does not work with 1.8.0-rc1. We need to cut a new release for this container, tag it, and update this field to something that works.

Reply:
@seanmwinn now there is `v0.6.0` tag that is compatible with 1.8.0-rc1/hubble-relay.